src/main/kotlin/com/cjbooms/fabrikt/util/KaizenParserExtensions.kt

```diff
@@ -222,6 +222,26 @@ object KaizenParserExtensions {
     fun Schema.isOneOfSuperInterface() =
         discriminator != null && discriminator.propertyName != null && oneOfSchemas.isNotEmpty()
 
+    private fun Schema.isPropertyWithAllOfSingleType() =
+        allOfSchemas?.size == 1 && isPropertySchema()
+
+    /**
+     * The `pathFromRoot` of a property schema ends with
+     * `/properties/<name of property>`, so we check if the
+     * penultimate segment is `properties`.
+     */
+    private fun Schema.isPropertySchema(): Boolean {
```

Comment:
Could this be used instead?
`isInlinedObjectDefinition()`

Reply:
`isInlinedObjectDefinition()` does not work for this case. `isObjectType()` returns false as the type of the schema `{"description":"The registered owner of the dog.","allOf":[{"$ref":"#/components/schemas/Person"}]}` is `null` and not `object`.

Would `isInlinedPropertySchema()` follow the naming convention or do you have something other in mind?

Reply:
maybe just `isInlinedSchema()`

Reply:
I renamed the function to `isInlinedPropertySchema()`.